var/spack/repos/builtin/packages/hipsycl/package.py

```diff
@@ -129,31 +137,34 @@ def filter_config_file(self):
             config = json.load(f)
         # 1. Fix compiler: use the real one in place of the Spack wrapper
         config["default-cpu-cxx"] = self.compiler.cxx
-        # 2. Fix stdlib: we need to make sure cuda-enabled binaries find
-        #    the libc++.so and libc++abi.so dyn linked to the sycl
-        #    ptx backend
-        rpaths = set()
-        so_paths = filesystem.find_libraries(
-            "libc++", self.spec["llvm"].prefix, shared=True, recursive=True
-        )
-        if len(so_paths) != 1:
-            raise InstallError(
-                "concretized llvm dependency must provide a "
-                "unique directory containing libc++.so, "
-                "found: {0}".format(so_paths)
+        if "+cuda" in self.spec:
+            # 2. Fix stdlib: we need to make sure cuda-enabled binaries find
```

Comment:
Note that everything after this is just indentation: I'm moving a cuda-specific action inside of an `if +cuda in spec` branch.